docs/browser.md

```diff
@@ -2,7 +2,7 @@
 id: browser
 title: Browser
 ---
-> KaTeX supports all major browsers, including Chrome, Safari, Firefox, Opera, Edge, and IE 9–11.
+> KaTeX supports all major browsers, including Chrome, Safari, Firefox, Opera, Edge, and IE 11.
```

Comment:
I think we can support IE 10, as there is only few caveats:
>If you have to support IE 10, the best solution is to *always* set an explicit `flex-shrink` value on all of your flex items, or to always use the longhand form (rather than the shorthand) in `flex` declarations to avoid the gotchas shown in the table above. Demo [6.1.a](https://codepen.io/philipwalton/pen/myOYqW) shows how not setting any flexibility properties causes an error, and demo [6.2.a](https://codepen.io/philipwalton/pen/zvvQdB) shows how using `flex: 1` can have the same problem.

Reply:
If you want to do the work it would take to support IE 10, feel free. I don't intend to waste my time chasing the whimsies of a browser with usage that is 0.06% and falling.

Reply:
In terms of how we support browsers we could look at tools like [autoprefixer](https://github.com/postcss/autoprefixer).  It has support for browserslist which we're already using.  As specific flexbox bugs in older browsers, that autoprefixer page suggests some possible [polyfills](https://github.com/postcss/autoprefixer#does-it-add-polyfills).  It looks like [postcss-preset-env](https://github.com/csstools/postcss-preset-env) does both auto-prefixing and polyfilling.  I'm not sure how difficult it would be set this up, but it would allow us to use flexbox without having to worry about the details on older browsers.

Reply:
> we could look at tools like ...

There's that pronoun again, "we".  If you want someone to research polyfills for IE 9 and IE 10, you'll have to find someone to do that. It won't be me.

Reply:
@ronkok sorry, I didn't mean "we" to imply "you".  I'll open a new issue for the autoprefixer/postcss-preset-env proposal.  I do think the autoprefixer would simplify parts of this PR, but I think this PR can go forward without it.